tests/integration/fixtures/relay.py

```diff
@@ -109,6 +107,8 @@ def inner(upstream, options=None, prepare=None, external=None):
             "internal": not external,
         }
 
+        process = background_process(RELAY_BIN + ["-c", str(dir), "run"])
```

Comment:
This was a rather curious race condition. We started the Relay process before telling mini_sentry about our Relay. This lead to spurious 403s if Relay started faster than we could proceed. Now, this is in the right order and there are no more 403s.